Index: standard/externals/Zend/Application/Bootstrap/BootstrapAbstract.php
===================================================================
--- standard/externals/Zend/Application/Bootstrap/BootstrapAbstract.php	(revision 623)
+++ standard/externals/Zend/Application/Bootstrap/BootstrapAbstract.php	(working copy)
@@ -349,6 +349,7 @@
                 if (0 === strcasecmp($resource, $pluginName)) {
                     return $this->_pluginResources[$pluginName];
                 }
+                continue;
             }
 
             if (class_exists($plugin)) {

