Index: standard/externals/Zend/Application/Module/Bootstrap.php
===================================================================
--- standard/externals/Zend/Application/Module/Bootstrap.php	(revision 488)
+++ standard/externals/Zend/Application/Module/Bootstrap.php	(working copy)
@@ -77,9 +77,10 @@
         }
 
         // ZF-6545: prevent recursive registration of modules
-        if ($this->hasPluginResource('modules')) {
-            $this->unregisterPluginResource('modules');
-        }
+//        if ($this->hasPluginResource('modules')) {
+//            $this->unregisterPluginResource('modules');
+//        }
+        //FIXME: some weird error here!
     }
 
     /**

