Index: Zend/Mail.php
===================================================================
--- Zend/Mail.php	(revision 21)
+++ Zend/Mail.php	(working copy)
@@ -522,7 +522,8 @@
     }
 
     /**
-     * Adds To-header and recipient
+     * Adds To-header and recipient. The $email may be specified in
+     * <code>"name-of-recipient" <to@email.com></code> format
      *
      * @param  string $email
      * @param  string $name
@@ -530,6 +531,12 @@
      */
     public function addTo($email, $name='')
     {
+    	// support for "name" <email> syntaxe
+    	if (preg_match('/^["](.*?)["]\s*?[<](.*?)[>]$/', trim($email), $matches)) {
+    		$email = $matches[2];
+    		$name = $matches[1];
+    	}
+    	
         $this->_addRecipientAndHeader('To', $email, $name);
         $this->_to[] = $email;
         return $this;
