Index: ../Zend/Loader/PluginLoader.php
===================================================================
--- ../Zend/Loader/PluginLoader.php	(revision 329)
+++ ../Zend/Loader/PluginLoader.php	(working copy)
@@ -125,6 +125,12 @@
     	if($prefix == "") {
     		return $prefix;
     	}
+    	
+        $last = strlen($prefix) - 1;
+        if ($prefix{$last} == '\\') {
+            return $prefix;
+        }
+
         return rtrim($prefix, '_') . '_';
     }
 
