From 24591f7e6696cc26ce3cf014ef3bd4952435a94b Mon Sep 17 00:00:00 2001
From: Steven Rosato <steven.rosato@majisti.com>
Date: Thu, 28 Oct 2010 01:53:05 -0400
Subject: [PATCH] Changed DELETE FROM to TRUNCATE

---
 .../Common/DataFixtures/Purger/ORMPurger.php       |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php b/lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php
index f043316..2dfd97f 100644
--- a/lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php
+++ b/lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php
@@ -83,7 +83,7 @@ class ORMPurger implements PurgerInterface
         }
 
         foreach($orderedTables as $tbl) {
-            $this->em->getConnection()->executeUpdate("DELETE FROM $tbl");
+            $this->em->getConnection()->executeUpdate("TRUNCATE $tbl");
         }
     }
 
@@ -125,4 +125,4 @@ class ORMPurger implements PurgerInterface
 
         return $associationTables;
     }
-}
\ No newline at end of file
+}
-- 
1.7.0.4

